resource/helpers/recipes-helper.py

```diff
@@ -104,13 +104,17 @@ def add_gpu_options_to_resp(response):
             'value': 'custom'
         }
     ]
-    response["gpu_memory_choices"] = [
+    response["gpu_memory_allocation_mode"] = [
         {
             'label': 'No limitation',
             'value': 'all'
         },
+        {
+            'label': 'Allocate only as much GPU memory as needed',
```

Comment:
I would advise a shorter label, to make it fit in the SELECT box. How about "Allocate memory automatically"?

Reply:
You are right, fixed in 2f16f21